src/fragments/lib/datastore/native_common/relational.mdx

```diff
@@ -1,94 +1,93 @@
+<Fragments fragments={{ js: js8 }} />
 
-import ios9 from "/src/fragments/lib/datastore/ios/relational/query-snippet.mdx";
+import ios9 from '/src/fragments/lib/datastore/ios/relational/query-snippet.mdx';
 
-<Fragments fragments={{ios: ios9}} />
+<Fragments fragments={{ ios: ios9 }} />
 
-import android10 from "/src/fragments/lib/datastore/android/relational/query-snippet.mdx";
+import android10 from '/src/fragments/lib/datastore/android/relational/query-snippet.mdx';
 
-<Fragments fragments={{android: android10}} />
+<Fragments fragments={{ android: android10 }} />
 
-import flutter11 from "/src/fragments/lib/datastore/flutter/relational/query-snippet.mdx";
+import flutter11 from '/src/fragments/lib/datastore/flutter/relational/query-snippet.mdx';
 
-<Fragments fragments={{flutter: flutter11}} />
+<Fragments fragments={{ flutter: flutter11 }} />
 
 ## Deleting relations
 
-When you delete a parent object in a one to many relationship, the children will also be removed from the DataStore and mutations for this deletion will be sent over the network. For example the following operation would remove the Post with id *123* as well as any related comments:
+When you delete a parent object in a one to many relationship, the children will also be removed from the DataStore and mutations for this deletion will be sent over the network. For example the following operation would remove the Post with id _123_ as well as any related comments:
 
-import js12 from "/src/fragments/lib/datastore/js/relational/delete-snippet.mdx";
+import js12 from '/src/fragments/lib/datastore/js/relational/delete-snippet.mdx';
 
-<Fragments fragments={{js: js12}} />
+<Fragments fragments={{ js: js12 }} />
 
-import ios13 from "/src/fragments/lib/datastore/ios/relational/delete-snippet.mdx";
+import ios13 from '/src/fragments/lib/datastore/ios/relational/delete-snippet.mdx';
 
-<Fragments fragments={{ios: ios13}} />
+<Fragments fragments={{ ios: ios13 }} />
 
-import android14 from "/src/fragments/lib/datastore/android/relational/delete-snippet.mdx";
+import android14 from '/src/fragments/lib/datastore/android/relational/delete-snippet.mdx';
 
-<Fragments fragments={{android: android14}} />
+<Fragments fragments={{ android: android14 }} />
 
-import flutter15 from "/src/fragments/lib/datastore/flutter/relational/delete-snippet.mdx";
+import flutter15 from '/src/fragments/lib/datastore/flutter/relational/delete-snippet.mdx';
 
-<Fragments fragments={{flutter: flutter15}} />
+<Fragments fragments={{ flutter: flutter15 }} />
 
 However, in a many to many relationship the children are not removed and you must explicitly delete them.
 
 ### Many-to-many
 
 For many-to-many relationships, you can use the `@manyToMany` directive and specify a `relationName`. Under the hood, Amplify creates a join table and a one-to-many relationship from both models.
 
+<Callout warning>
+
+Join table records must be deleted prior to deleting the associated records
```

Comment:
@renebrandel this is newly added since your last review

Reply:
I like the call out but I think we should have an example. "For example a Post has many Authors. etc. etc. Delete the PostAuthor join table from the relationship first..."

Reply:
Will update

Reply:
Done